feat: Add Flagsmith provider (#89)
open-feature/dotnet-sdk-contrib: src/OpenFeature.Contrib.Providers.Flagsmith/README.md

> # Flagsmith Feature Flag .NET Provider

Reviewer: ```suggestion
# Flagsmith .NET Provider
```

> The Flagsmith Flag provider allows you to connect to your Flagsmith instance.  

Reviewer: ```suggestion
The Flagsmith provider allows you to connect to your Flagsmith instance through the OpenFeature SDK
```

# .Net SDK usage

## Install dependencies

The first things we will do is install the **Open Feature SDK** and the **Flagsmith Feature Flag provider**.

### .NET Cli
```shell
dotnet add package OpenFeature.Contrib.Providers.Flagsmith
```
### Package Manager

```shell
NuGet\Install-Package OpenFeature.Contrib.Providers.Flagsmith
```
### Package Reference

```xml
<PackageReference Include="OpenFeature.Contrib.Providers.Flagsmith" />
```
### Packet cli

```shell

> paket add OpenFeature.Contrib.Providers.Flagsmith

Reviewer: ```suggestion
packet add OpenFeature.Contrib.Providers.Flagsmith
```
This looks like a typo but I'm not familiar with packet.
